compiler/codegen/codegen.cpp

```diff
@@ -2052,6 +2055,9 @@ codegen_config() {
 
         args[3] = info->irBuilder->getInt32(var->hasFlag(FLAG_PRIVATE));
 
+        args[4] = info->irBuilder->getInt32(var->hasFlag(FLAG_DEPRECATED));
+        args[5] = info->irBuilder->CreateLoad(new_CStringSymbol(var->getDeprecationMsg())->codegen().val);
```

Comment:
When you were getting LLVM errors last week, was it due to not having these lines and the ones just above?  Or something else?  (just curious how that resolved).

Reply:
Yeah, it was due to not having these lines